cmd/prometheus-config-reloader/main.go

```diff
@@ -76,6 +77,11 @@ func main() {
 		"address on which to expose metrics (disabled when empty)").
 		String()
 
+	webConfig := app.Flag(
+		"web-config-file",
+		"Path to configuration file that can enable TLS or authentication. See: https://prometheus.io/docs/prometheus/latest/configuration/https/",
```

Comment:
I'm wondering if worth to mention this is experimental, since Prometheus is warning about this as well.

Reply:
Thanks you for you review. Experimental warning added.